scss/modules/_infobox.scss

```diff
@@ -7,6 +7,14 @@
       text-align: center;
     }
 
+    .symbol {
```

Comment:
nogo - copy css block - we should move the current one (remove nesting) `.symbol` is now global anyway